Header header
int8 flight_stage